theano/sandbox/rng_mrg.py

```diff
@@ -441,8 +444,9 @@ def c_code(self, node, name, inp, out, sub):
 
         for (int i = 0; i < %(ndim)s; ++i)
         {
-            odims[i] = *(dtype_%(size)s *)PyArray_GETPTR1(%(size)s, i);
-            n_elements *= odims[i];
+            odims_i = *(dtype_%(size)s *)PyArray_GETPTR1(%(size)s, i);
+            odims[i] = odims_i;
```

Comment:
should we test that odims_i fix in  npy_intp? I'm not sure how useful it is to check that.

Otherwise, I'm ok to merge after travis passed if you an all mrg tests.